backend/stackexchange/stack_api.py

```diff
@@ -0,0 +1,71 @@
+from stackapi import StackAPI
+import json
+from dataclasses import dataclass
+from dacite import from_dict
+
+
+class ApiResponse:
+    """ Gets the users identified in ids in {ids}.
+    Typically this method will be called to fetch user profiles when you have obtained user ids from some other source, such as /questions.
+    {ids} can contain up to 100 semicolon delimited ids. To find ids programmatically look for user_id on user or shallow_user objects.
+    The sorts accepted by this method operate on the following fields of the user object:
+    reputation – reputation
+    creation – creation_date
+    name – display_name
+    modified – last_modified_date
+    reputation is the default sort.
+    It is possible to create moderately complex queries using sort, min, max, fromdate, and todate.
+    This method returns a list of users. """
+
+    def __init__(self, site: str, user_id: str, order: str = 'desc', sort: str = 'name') -> None:
+        self._site = StackAPI(site)
+        self.order = order
+        self.sort = sort
+        self._api_url = f'users/{user_id}'
+        self._data_all = self._site.fetch(self._api_url, order=self.order, sort=self.sort)
+
+    def print_api_response_json(self):
+        print(json.dumps(self._data_all, indent=4))
+
+    @property
```

Comment:
["@property; pythonic way to use getters and setters." ](https://www.programiz.com/python-programming/property) - but it's not getter or setter. :)